AUTHORS_GRANTING_LGPL_LICENSE.txt

```diff
@@ -0,0 +1,16 @@
+VOLK is going to migrating from GPLv3 (GNU General Public license version 3.0)
+to LGPLv3 (GNU Lesser General Public License version 3.0).
+
+This file is a list of the authors who agreed to grant an LGPL license to the
+code they contributed to this repository. In case the affected code is currently
+licensed differently (GPLv3), this gives the right to use the current
```

Comment:
"licensed differently (GPLv3)" -> "licensed differently than VOLK's current license (GPLv3)"

Reply:
I assume the interesting part in our license is 
https://github.com/gnuradio/volk/blob/237a6fc9242ea8c48d2bbd417a6ea14feaf7314a/COPYING#L563

Followed by our license statement:
https://github.com/gnuradio/volk/blob/main/README.md#license
> VOLK is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation; either version 3, or (at your option) any later version.

With these things in mind, I thought VOLK is currently licensed under GPLv3+ (or later) and I'd assume we want to re-license to LGPLv3+.

Finally, the [GREP](https://github.com/gnuradio/greps/pull/33/files) currently states:

> The proposed new license is [LGPLv3+](https://www.gnu.org/licenses/lgpl-3.0.html).

We should be consistent. I'd argue that's the most important part.